### ISSUE TITLE
Send to EDDN is broken in `EDMC.py -n`

### ISSUE THREAD

**Issue by Athanasius** [EDCD/EDMarketConnector#1720]
Whilst this is now *more* broken in forthcoming 5.6.0, it was already broken, in a different manner, in 5.5.0.

5.5.0:
```
C:\Program Files (x86)\EDMarketConnector>EDMC.exe -n
HIP 10792,Crimson Exchange
2022-11-28 15:12:09.720 UTC - ERROR - 8752:12824:12824 EDMC.main:441: Failed to send data to EDDN
Traceback (most recent call last):
  File "EDMC.py", line 436, in main
TypeError: EDDN.export_commodities() takes 3 positional arguments but 4 were given
```

5.6.0-beta1:
```
C:\Program Files (x86)\EDMarketConnector>EDMC.exe -n
2022-11-28 15:12:56.442 UTC - INFO - 11828:8508:8508 monitor.EDLogs.populate_version_info:1707: Parsed self.state['GameVersion']='4.0.0.1450' into self.version_semantic=Version('4.0.0+1450')
2022-11-28 15:12:56.443 UTC - INFO - 11828:8508:8508 monitor.EDLogs.populate_version_info:1707: Parsed self.state['GameVersion']='4.0.0.1450' into self.version_semantic=Version('4.0.0+1450')
HIP 10792,Crimson Exchange
2022-11-28 15:12:57.337 UTC - INFO - 11828:8508:8508 plugins.eddn.EDDNSender.convert_legacy_file:258: Converson` to `eddn_queue-v1.db` complete, removing `replay.jsonl`
2022-11-28 15:12:57.339 UTC - ERROR - 11828:8508:8508 EDMC.main:441: Failed to send data to EDDN
Traceback (most recent call last):
  File "EDMC.py", line 434, in main
  File "C:\Program Files (x86)\EDMarketConnector\plugins\eddn.py", line 601, in __init__
    self.sender = EDDNSender(self, self.eddn_url)
  File "C:\Program Files (x86)\EDMarketConnector\plugins\eddn.py", line 182, in __init__
    self.eddn.parent.after(self.eddn.REPLAY_STARTUP_DELAY, self.queue_check_and_send, True)
AttributeError: 'NoneType' object has no attribute 'after'
```

**Comment by Athanasius** [comment]
Now in released 5.7.0.

**Comment by Athanasius** [comment]
Now in released 5.7.0.
